engine/input/src/input.cpp

```diff
@@ -68,11 +68,20 @@ namespace dmInput
     HBinding NewBinding(HContext context)
     {
         Binding* binding = new Binding();
-        memset(binding, 0, sizeof(Binding));
         binding->m_Context = context;
+        binding->m_Actions.Clear();
```

Comment:
Since the Bindings struct has non-pod members, it's not ok to memset it, as it then overwrites the data of those members.